engine/language-server/src/main/scala/org/enso/languageserver/util/UnhandledLogging.scala

```diff
@@ -1,10 +1,13 @@
 package org.enso.languageserver.util
 
-import akka.actor.{Actor, ActorLogging}
+import akka.actor.Actor
+import akka.event.Logging
 
-trait UnhandledLogging { this: Actor with ActorLogging =>
+trait UnhandledLogging { this: Actor =>
+
+  private val akkaLogger = Logging(context.system, this)
 
   override def unhandled(message: Any): Unit =
-    log.warning("Received unknown message: {}", message)
+    akkaLogger.warning("Received unknown message: {}", message)
```

Comment:
Is this surely the right logger to use?

Because at first glance it seems to me like it may be the same one as in `ActorLogging` which we seem to need to avoid.

Reply:
Akka logger is used here to be able to control the logger verbosity. Since this is related to the Akka logic, this will be controlled by the Akka log level settings.
But you made me think if we can use the SLF4J logger here

Reply:
From my understanding it was that the Akka logging will not handle masking correctly.

But your point also makes sense. Moreover - I think this `message` of type `Any` will not trigger the masking mechanism anyway, right? Then maybe it can stay as is, unless we are worried that the message may contain PII.

Reply:
If the message was designed to be mask (extended the `ToLogString`, it will be masked. But I changed the function to log just the class to be extra safe, or if the message is extremely big.